src/queries/getSwapInfo.ts

```diff
@@ -98,25 +98,24 @@ export default async function getSwapInfo(nativeId: string) {
     throw new UnrecoverableError('Deposit amount is missing');
   }
 
-  const { sourceChain, sourceAsset, destinationAsset, completedEventId } = swap;
+  const { sourceChain, sourceAsset, destinationAsset, completedEventId, fokMinPriceX128 } = swap;
   const depositChannelCreationTimestamp = swap.swapChannel?.issuedBlockTimestamp;
   const depositTimestamp = swap.depositBlock?.stateChainTimestamp;
   const preDepositBlockTimestamp = swap.preDepositBlock?.stateChainTimestamp;
   const egressTimestamp = swap.egress?.scheduledEvent.block.timestamp;
-  const fokMinPriceX128 = swap.swapChannel?.fokMinPriceX128;
   const completedAt = swap.completedEvent?.block.timestamp;
 
   const egressValueUsd = toUsdAmount(swap.egress?.valueUsd);
-  const broker = swap.swapChannel?.broker.account;
-  const numberOfChunks = swap.numberOfChunks ?? 1;
+  const broker = swap.broker?.account;
+  const numberOfChunks = swap.dcaNumberOfChunks;
```

Comment:
```suggestion
  const numberOfChunks = swap.dcaNumberOfChunks ?? 1;
```

Reply:
Or is that the default value in the db?

Reply:
yes, the column  is not nullable and defaults to 1 🙂